src/reference/00-Getting-Started/07A-Task-Graph.md

```diff
@@ -0,0 +1,324 @@
+---
+out: Task-Graph.html
+---
+
+  [Basic-Def]: Basic-Def.html
+  [Scopes]: Scopes.html
+  [Make]: https://en.wikipedia.org/wiki/Make_(software)
+  [Ant]: http://ant.apache.org/
+  [Rake]: https://ruby.github.io/rake/
+
+Task graph
+----------
+
+This page explains `build.sbt` definition in more detail.
+It assumes you've read [.sbt build definition][Basic-Def] and
+[scopes][Scopes].
```

Comment:
I agree with @briantopping, but I'm not sure how/where would you say that so that it satisfies the "Makefile-literate" without sounding too arcane to everybody else.